setup/MySQL/convert_mysql.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#Set database name
+if [ $# -eq 2 ]; then
+   MYDATABASE="atpdatabase"
+else
+   MYDATABASE=$3
+fi
+
+#Get MySQL username and password from the arguments
+USERNAME=$1
+PASSWORD=$2
+
+#Get PWD
+PWD=`pwd`
+
+echo "Creating $MYDATABASE"
+
+mysql -u $USERNAME -p$PASSWORD << EOF
+
+#Create database from scratch
```

Comment:
Could you also add a `drop database if exists` line? Apart from setup, this script should also be usable for resetting or updating the DB when new CSVs are added.